src/Aspire.Hosting.Azure/AzureBicepResource.cs

```diff
@@ -47,12 +47,12 @@ public AzureBicepResource(string name, string? templateFile = null, string? temp
     /// <summary>
     /// Outputs that will be generated by the bicep template.
     /// </summary>
-    public Dictionary<string, object?> Outputs { get; } = [];
+    public Dictionary<string, object?> Outputs { get; } = new Dictionary<string, object?>(StringComparer.OrdinalIgnoreCase);
```

Comment:
There's a really spooky bug with provisioning ACA environments where the outputs returned in the ARM payload have inconsistent casing (e.g. `azurE_CONTAINER_REGISTRY_NAME`) which prevents parameters from being filled correctly by the BicepProvisioner. I am fixing this by ignoring casing for output names by default.

Reply:
It's ARM, in azd we handle this by doing toUpper to all outputs names